packages/rest/src/__tests__/integration/rest.application.integration.ts

```diff
@@ -145,8 +145,44 @@ describe('RestApplication (integration)', () => {
     });
   });
 
+  it('controls redirect routes', async () => {
+    givenApplication();
+    restApp.controller(DummyController);
+    restApp.redirect('/page/html', '/html');
+    await restApp.start();
+    client = createRestAppClient(restApp);
+    const response = await client.get('/page/html').expect(303);
```

Comment:
Please verify the value of `Location` header too.

Reply:
test replaced with custom status code 
similarly here i make new request to the redirect route to check the result 
based on location header no need to check  the header manually 
the request will check that if its fail it means the location is not sett correctly